examples/management/get-devices.mjs

```diff
@@ -0,0 +1,35 @@
+/**
+ * Example that loads all devices for a given workspace.
+ */
+import { config } from "dotenv";
+import path from "path";
+import axios from "axios";
+import { loadAllEntities, isRunDirectly } from "../utils.mjs";
+config({
+  path: process.env.CONFIG_PATH || path.resolve(process.cwd(), ".env"),
+});
+
+const apiKey = process.env.API_KEY;
+const workspaceId = process.env.WORKSPACE_ID;
+const baseUrl = process.env.BASE_URL || "https://api.akenza.io";
+
+export async function loadDevices(pageNumber, pageSize = 100) {
```

Comment:
maybe also assign a default value to `pageNumber`, also add a comment that paging is 0-based

Reply:
I have added a task to describe pagination in the api docs in the next sprint